spec.html

```diff
@@ -24080,7 +24080,6 @@ <h1>Runtime Semantics: GetModuleNamespace ( _module_ )</h1>
         </emu-note>
       </emu-clause>
 
-      <!-- es6num="15.2.1.20" -->
```

Comment:
Can you elaborate on why these are being deleted?

Reply:
Sure: PR #1237

Reply:
touché :-p